Android/app/src/androidTest/java/io/github/project_travel_mate/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,21 @@
+package io.github.project_travel_mate;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static junit.framework.TestCase.assertEquals;
+
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() throws Exception {
```

Comment:
why is there need to add this test for this PR?

Reply:
Just replaced the deprecated Test which was there before.

Reply:
Dont do any other change thats needed apart for the issue. Revert it